platform/android/java/lib/src/org/godotengine/godot/GodotIO.java

```diff
@@ -306,6 +309,19 @@ public int getScreenOrientation() {
 		}
 	}
 
+	public int getDisplayRotation() {
+		WindowManager windowManager = (WindowManager)activity.getSystemService(Context.WINDOW_SERVICE);
+		int rotation = windowManager.getDefaultDisplay().getRotation();
+		if (rotation == Surface.ROTATION_90) {
+			return 270;
+		} else if (rotation == Surface.ROTATION_180) {
+			return 180;
+		} else if (rotation == Surface.ROTATION_270) {
+			return 90;
+		}
```

Comment:
Is there a reason `270` and `90` are inverted?

Reply:
The `rotation` value tells us how much the display is rotated from natural. We calculate the rotation to counteract that, so that when combined with the rest of the camera processing, the camera feed's orientation matches the display.
```
void CameraFeedAndroid::set_rotation() {
	int display_rotation = DisplayServerAndroid::get_singleton()->get_display_rotation();

	int sign = position == CameraFeed::FEED_FRONT ? 1 : -1;
	float imageRotation = (orientation - display_rotation * sign + 360) % 360;
	transform.set_rotation(real_t(Math::deg_to_rad(imageRotation)));
}
```

Reply:
Let's move that calculation within `CameraFeedAndroid`; that'll allow other components in the codebase to use `getDisplayRotation()` without having to reverse the logic here.